app.js

```diff
@@ -1,8 +1,19 @@
-const express = require ('express')
-const { getApi } = require('./nc_controllers/app.controller')
+const express = require("express");
+const { getApi, getTopics } = require("./nc_controllers/app.controller");
+const {
+  psqlErrorHandler,
+  customErrorHandler,
+} = require("./nc models/error-handler");
 
-const app = express()
+const app = express();
 
-app.get('/api', getApi)
+app.get("/api", getApi);
 
-module .exports = app
\ No newline at end of file
+app.get("/api/topics", getTopics);
+app.all("*", (req, res) => {
+  res.status(404).send({ message: "not found" });
+});
+app.use(psqlErrorHandler);
```

Comment:
With regards to Test DRIVEN Development, do we need to handle any psql errors yet?